cmd/kubelet/app/server.go

```diff
@@ -353,8 +353,14 @@ func run(s *options.KubeletServer, kubeDeps *kubelet.KubeletDeps) (err error) {
 				if err != nil {
 					return err
 				}
+			} else {
+				glog.Warningf("failed to init kubelet configure sync : %v", err)
```

Comment:
nit: can you reword this to `failed to init dynamic Kubelet configuration sync: %v`, and make it an error instead of a warning?